org.eclipse.winery.bpmn4tosca.converter.tobpel/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Tue Jun 27 19:00:17 CEST 2017
```

Comment:
Do not re-add gradle.

We changed from gradle tp Maven also in this submodule.

Maybe, it would be a good idea, just to add your changes to the test class to the commit and throw away the other changes.

1. git gui
2. add files you changed
3. commit
4. `git reset --hard` on the command line